operations/System_Recovery.md

```diff
@@ -574,4 +576,57 @@ The latest CSM documentation has been installed on the master nodes. See [Check
 
    See [Nexus Export and Restore](package_repository_management/Nexus_Export_and_Restore.md)
 
+### Slurm
+
+To restore Slurm data from backup, follow sections
+*10.3.11 Restore Slurm Accounting Database from Backup* and
+*10.3.13 Restore Slurm Spool Directory from Backup* in the
+**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+
+After restoring Slurm data from backup, check that the procedure was successful.
+
+1. (`uan#`) Check the accounting records.
+
+   ```bash
+   sacct -S <date>
+   ```
+
+1. (`uan#`) Check queued jobs.
+
+    ```bash
+   squeue
```

Comment:
If the backup was not successful, would `squeue` be empty?  Is there an example that can be added to show what might be seen if the restore was not successful?  Same question for `sinfo `below` as well as the pbs commands.

Reply:
The queue should match the state when it was backed up. So if the queue was empty, it should still be empty. The only way you could tell that a restore failed would be if there were jobs in the queue when it was backed up, but they don't show up after the backup/restore.